recipes/fmt/all/conanfile.py

```diff
@@ -81,6 +82,14 @@ def package_id(self):
     def validate(self):
         if self.settings.get_safe("compiler.cppstd"):
             check_min_cppstd(self, 11)
+        if Version(self.version).major in (8, 10, 11) and \
+             self.settings.compiler == "clang" and Version(self.settings.compiler.version) >= "20" and \
+             Version(str(self.settings.compiler.cppstd).replace("gnu", "")) >= "20":
```

Comment:
This can probably be simplified with https://docs.conan.io/2/reference/tools/build.html#conan-tools-build-valid-min-cppstd

Reply:
🤦

Reply:
@AbrilRBS Thank you! Address by the commit dd53efb50d6568695c1b8be6235445a150bbb8c5